.github/labeler.yml

```diff
@@ -0,0 +1,35 @@
+"group: ansible":
+  - changed-files:
+      - any-glob-to-any-file: 'ansible/*'
+
+"group: dns":
+  - changed-files:
+      - any-glob-to-any-file: 'dns/*'
+
+"group: docs":
+  - changed-files:
+      - any-glob-to-any-file: 'docs/*'
+
+"group: github actions":
+  - changed-files:
+      - any-glob-to-any-file: '.github/workflows/*'
+
+"group: kubernetes":
+  - changed-files:
```

Comment:
Do these labels need to exist before it can create them? Or. Will it add them auto?

Reply:
It will create them